web/static/js/components/stage_progression_button.jsx

```diff
@@ -33,11 +36,15 @@ export class StageProgressionButton extends Component {
       buttonDisabled,
       className,
       currentUser,
+      presences,
       retroUpdateRequested,
       config,
+      votes,
     } = this.props
 
-    const { modalOpen, showSubmitIdeaPrompt } = this.state
+    const { modalOpen } = this.state
+    const showAllVotesInPrompt = !isEmpty(presences)
```

Comment:
We are determining we are on the correct the stage (voting stage) here by the lack of presences. Would this be better determined in another way?

Reply:
I think this introduces a bug, in that the tooltip would show in all stages _following_ the voting stage.

Thankfully, we can scope this to the voting stage, as we have the `stage` available in the redux store (under the `retro` key/reducer), so if we pass the stage in the `connect` call, that simplifies this logic and scopes the tooltip to this stage alone.